_topic_maps/_topic_map.yml

```diff
@@ -1349,18 +1349,12 @@ Topics:
       File: aws-load-balancer-operator-release-notes
     - Name: AWS Load Balancer Operator in OpenShift Container Platform
       File: understanding-aws-load-balancer-operator
+    - Name: Preparing an AWS STS cluster for the AWS Load Balancer Operator
+      File: preparing-sts-cluster-for-albo
     - Name: Installing the AWS Load Balancer Operator
       File: install-aws-load-balancer-operator
-    - Name: Installing the AWS Load Balancer Operator on a cluster that uses AWS STS
-      File: installing-albo-sts-cluster
-    - Name: Creating an instance of the AWS Load Balancer Controller
-      File: create-instance-aws-load-balancer-controller
-    - Name: Serving multiple ingress resources through a single AWS Load Balancer
-      File: multiple-ingress-through-single-alb
-    - Name: Adding TLS termination
-      File: add-tls-termination
-    - Name: Configuring cluster-wide proxy
-      File: configure-egress-proxy-aws-load-balancer-operator
```

Comment:
Since the assemblies are no longer being used anymore (instead, their modules are being re-used elsewhere, as I understand it), the assembly files can be fully `git rm`'d from the repo. Not blocking merge, we can clean up in a follow-up PR.